src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -243,7 +243,7 @@ protected void addHighLevelBindings(final DocWorkUnit workUnit)
      */
     protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         final String tagFilterPrefix = getTagPrefix();
-        Arrays.stream(currentWorkUnit.getClassDoc().inlineTags())
+        Arrays.stream(currentWorkUnit.getClassDoc().tags())
```

Comment:
Actually that's a normal javadoc processing feature. These kind of tags (`@see`, `@author`, etc.) are suppose to be at the end of the javadoc instead of embedded in the text. That's why embedded tags such as `@link` have the curly-brackets syntax.

I think that the fix is to point out that this is like that... and actually IntelliJ re-format the javadoc putting them at the end, so I think that it's fair enough to keep it as it is now...